content/articles/creating-dummy-data-in-python-using-faker-package/index.md

```diff
@@ -0,0 +1,99 @@
+### Introduction
+For various reasons, it's essential to have phoney data on hand. In this post, dummy data is generated using the Python Faker library. It's critical to test and evaluate software and hardware with dummy data. It's possible to test your code by running it through various scenarios. Faked data can be easily generated with the Python function Faker.
+
+For testing and operational reasons, dummy data is used. Your code can be tested against a variety of different types of stimuli.
+### Prerequisites
+1. Before proceeding with this tutorial, you should know the python programming language.
+2. Have a python IDE or follow this [documentation](https://www.python.org/) to install.
+
+> Setup Faker with the pip command:
+
+```Python
+Pip install Faker
+```
+
+### Listed below are methods and types of dummy data
+#### 1. Faker generators can be created and initialized

```

Comment:
This is not a method right? It's common to all of the them. So, don't add it as a subheading